x/concentrated-liquidity/position.go

```diff
@@ -682,11 +682,11 @@ func (k Keeper) RemovePositionIdToLock(ctx sdk.Context, positionId, underlyingLo
 // If lock is no longer active, returns false.
 func (k Keeper) PositionHasActiveUnderlyingLock(ctx sdk.Context, positionId uint64) (hasActiveUnderlyingLock bool, lockId uint64, err error) {
 	// Get the lock ID for the position.
+	// The only possible error retuned from `GetLockIdFromPositionId` is when position id to lock id mapping is not present.
+	// In this case, we silence the error and return false instead.
```

Comment:
This comment should be removed now

Reply:
fixed in https://github.com/osmosis-labs/osmosis/pull/5135/commits/d03e84e1e998c365c29e1f89b39e3d0003287dff